xbmc/Application.cpp

```diff
@@ -1081,10 +1081,30 @@ bool CApplication::InitDirectoriesLinux()
   else
     userHome = "/root";
 
+  /* Get XDG paths.
+   * Though a slash is expected at the end anyway, let's cover ourselves, just in case. */
+  std::string xdgDataPath;
+  if (getenv("XDG_DATA_HOME"))
+  {
+    xdgDataPath = getenv("XDG_DATA_HOME");
+    URIUtils::AddSlashAtEnd(xdgDataPath);
```

Comment:
unfortunately. nope. it becomes /storage/kodi. I need /storage/.kodi (with a dot) for all OE specific binary stuff that I am not going to fix again. but well it's ok it would be a few liner partial revert I will take care...